backend/index.js

```diff
@@ -1,9 +1,21 @@
 const express = require('express');
 const app = express();
+const PORT = 3000;
 const cors = require('cors')
+const userRouter = require('./routes/user')
 
 //middleware
 app.use(express.json())
 
 // enables cors for all routes
 app.use(cors())
+
+// user routes
+app.use('/user', userRouter)
+
+// main route
+
+app.get('/', (req, res) => {
+    res.send('Backend of Fina!')
+});
+
```

Comment:
did you missing the listen method?

```suggestion
app.listen(PORT, () => {
  console.log(`Server running on port ${PORT}`);
});
```